onnx/version_converter/helper.cc

```diff
@@ -0,0 +1,79 @@
+// Helper Methods for Adapters
+
+#include "onnx/version_converter/helper.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+    bool onnx_opset7_requires_broadcasting(int64_t axis,
+        const std::vector<Dimension>& input1_sizes,
+        const std::vector<Dimension>& input2_sizes) {
+      // Assert that broadcasting semantics are correct
```

Comment:
Can you check that all the dimensions use int (dim) instead of param? Otherwise you may have 0 vs 0 cases.